tests/integration/sumcoins/main.py

```diff
@@ -1,5 +1,37 @@
 import argparse, redis, json, msgpack
 from enum import Enum
+from build.types_pb2 import PBWallet, PBNetworkStats
+
+class ProtoMsg(object):
```

Comment:
Why did you define an entire class like this? As to match my current code or so? Seems like I can just use

```python
wallet = rtypes.types_pb2.PBWallet()
wallet.ParseFromString(b)
```

And get a nice struct with property members that I can use. Doesn't seem to play well with linting, but does compile and run fine.